types/response.go

```diff
@@ -24,9 +25,14 @@ func SerializeHTTPResponse(
 	// We have to avoid using http.Header.Get(key) because it only returns the
 	// first value of the key.
 	for key := range response.Header {
+		// Sort the header values to ensure that the order of the values is
```

Comment:
Is this something that'll require on-chain enforcement too?

Reply:
Since on-chain code doesn't inspect or check the content of `RelayResponse.Payload`, we don't need on-chain enforcement.

It's just that now we have deterministic bytes result out of a backend service response.